loyalty_partner_applicability/models/loyalty_partner_applicability_mixin.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Tecnativa - Pilar Vargas
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+from odoo.osv import expression
+from odoo.tools.safe_eval import datetime, safe_eval
+
+
+class LoyaltyPartnerApplicabilityMixin(models.AbstractModel):
+
+    _name = "loyalty.partner.applicability.mixin"
+    _description = "Manage the applicability of loyalty elements to partners"
+
+    partner_ids = fields.Many2many(
+        string="Specific customers",
```

Comment:
It is best to avoid ‘Specific customers’ because it is ambiguous and does not clearly reflect that the field limits the promotion to certain customers, which may confuse the user about its purpose.

```suggestion
        string="Allowed partners",
```